hack/source-manifests.sh

```diff
@@ -71,7 +71,7 @@ gen_args="generate csv --csv-version=$TMP_CSV_VERSION --output-dir=$OUTDIR_TEMPL
 if [ -n "$CSV_REPLACES_VERSION" ]; then
 	gen_args="$gen_args --from-version=$CSV_REPLACES_VERSION"
 fi
-$OPERATOR_SDK $gen_args
+$OPERATOR_SDK "$gen_args"
```

Comment:
This change breaks `make gen-latest-csv` when running on darwin (macos) 

The error is:
`Error: unknown command "generate csv --csv-version=9999.9999.9999 --output-dir=build/_output/csv-templates" for "operator-sdk"`

Removing the quotes (reverting the change) fix the issue